### PR TITLE
update filers documentation

### PR DESCRIPTION
**Pull request by jpkha**

### CODE REVIEW

File: docs/reference/filters.md [hunk 1]
```diff
@@ -107,11 +107,15 @@ The available operators are:
...
 * `nil` for 'empty', and `not_nil` for non-empty (any value),
   * Note that `nil` and `not_nil` are the only operators that do not require anything inside `values`.
+* `starts_with` / `not_starts_with` / `ends_with` / `not_ends_with` / `contains` / `not contains` are available for searching in short string fields (name, value, title...).
+* `search`  is available in short string and text fields.
 
 When using a numerical comparison operators (`gt` and the like) against textual values, the alphabetical ordering is used.
 
 Some operator may not be allowed for some key, for additional information please navigate to the "Special keys" section.
 
+There is a small differences between `search` and `contains`. `search` finds any occurrence of specified words, regardless of order, while "contains" specifically looks for the exact sequence of words you provide.
```
Comment (yassine-ouaamou) [inline docs/reference/filters.md:117]: a small difference
no "s"

File: docs/reference/filters.md [hunk 2]
```diff
@@ -107,11 +107,15 @@ The available operators are:
 * `lt` / `lte` for 'lower than' / 'lower than or equal',
 * `nil` for 'empty', and `not_nil` for non-empty (any value),
   * Note that `nil` and `not_nil` are the only operators that do not require anything inside `values`.
+* `starts_with` / `not_starts_with` / `ends_with` / `not_ends_with` / `contains` / `not contains` are available for searching in short string fields (name, value, title...).
+* `search`  is available in short string and text fields.
 
 When using a numerical comparison operators (`gt` and the like) against textual values, the alphabetical ordering is used.
```
Comment (Archidoit) [inline docs/reference/filters.md:113]: maybe write this precision just after the 'gt' line

File: docs/reference/filters.md [hunk 3]
```diff
@@ -107,11 +107,15 @@ The available operators are:
...
 
 Some operator may not be allowed for some key, for additional information please navigate to the "Special keys" section.
 
+There is a small difference between `search` and `contains`. `search` finds any occurrence of specified words, regardless of order, while "contains" specifically looks for the exact sequence of words you provide.
```
[the remark this answers is not in the document]
Comment (jpkha) [reply docs/reference/filters.md:117]: Yes I will do that.

File: docs/reference/filters.md (hunk 2 above)
Comment (jpkha) [reply docs/reference/filters.md:113]: Not sure about understanding this point ?

**Pull request merged**